server/routes/api/index.js

```diff
@@ -1,8 +1,11 @@
 import express from 'express';
 import userRoutes from './users';
+import Token from '../../helpers/Token';
+import UserController from '../../controllers/users';
 
 const apiRoutes = express.Router();
 
-apiRoutes.use('/', userRoutes);
+apiRoutes.use('/auth', userRoutes);
+apiRoutes.use('/users/:followid/follow', Token.authorize, UserController.follow);
```

Comment:
??????????????????????????????????????????????????????????????????????????????????????????????????????

This should have been `apiRoutes.post ...`

However, 
create a new router file `profileRoutes`
change this to 
`apiRoutes.use('/profiles', Token.authorize, <new profile router>);`

Then in this new profile router define this as:

`profileRoutes.post('/:username/follow, UserController.follow);`

Reply:
Thanks. I will implement this now.